src/Adapter/Cart/CommandHandler/RemoveProductFromCartHandler.php

```diff
@@ -39,21 +43,47 @@
  */
 final class RemoveProductFromCartHandler extends AbstractCartHandler implements RemoveProductFromCartHandlerInterface
 {
+    /**
+     * @var ContextStateManager
+     */
+    private $contextStateManager;
+
+    /**
+     * @param ContextStateManager $contextStateManager
+     */
+    public function __construct(
```

Comment:
I thinks that's a BC break

Reply:
It looks like this class didn't exist before 1.7.7 it should be fine 👍

Reply:
oops, no, that's a 1.7.7.1 PR, it's a bc break then...

Reply:
:+1: Done